airbyte/_processors/sql/base.py

```diff
@@ -908,12 +899,14 @@ def _emulated_merge_temp_table_to_final_table(
             conn.execute(update_stmt)
             conn.execute(insert_new_records_stmt)
 
-    @final
     def _table_exists(
         self,
         table_name: str,
     ) -> bool:
-        """Return true if the given table exists."""
+        """Return true if the given table exists.
```

Comment:
This was super slow for bigquery. I overrode this with a custom bigquery implementation which uses a different API to check for table existence.